rwiki/rwiki-impl/impl/src/bundle/uk/ac/cam/caret/sakai/rwiki/component/bundle/Messages_eu.properties

```diff
@@ -132,3 +132,6 @@ XLSTChangesHandler.23=Azken aldaketak hemen\:
 PreMacro.1=Edukia honela errendatzen du
 CommentMacro.0=Makro bat da linean iruzkin bat sortzen duena callout-ekin 
 CommentMacro.1=ezein ez
+
+### S2U-39 5.2.3.1 Wiki: Add group support to Wiki pages
+availableTo.Groups=Orrialde hau taldeentzat bakarrik dago erabilgarri: 
```

Comment:
@pablosanroman Could you please help with the translation?

Reply:
Hi @mpellicer . 
availableTo.Groups=Orrialde hau taldeentzat baino ez dago eskuragarri: